src/server/process_request.c

```diff
@@ -1517,7 +1517,7 @@ free_br(struct batch_request *preq)
 			break;
 		case PBS_BATCH_DeleteJobList:
 			if (preq->rq_ind.rq_deletejoblist.rq_jobslist)
-				free(preq->rq_ind.rq_deletejoblist.rq_jobslist);
+				free_string_array(preq->rq_ind.rq_deletejoblist.rq_jobslist);
```

Comment:
The difference between free() and free_string_array() will be a leak, not invalid read/writes.  How does this fix the invalid read/writes?

Reply:
I have tried to rerun the test which found the invalid read/write and memory leaks, seeks memory issues were not reported in valgrind. Let me try multiple times and confirm the test results. Meanwhile, let mark PR as WIP.